src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+export type SortFunc = (a: Student[], b: SortType, c: SortOrder) => Student[];
 
+export const sortStudents: SortFunc = (students, sortBy, order) => {
+  type Num = number;
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
-}
+  const sortCallback = (a: Student, b: Student): Num => {
+    const studentA: Student = order === 'asc'
+      ? a
+      : b;
+
+    const studentB: Student = order === 'asc'
+      ? b
+      : a;
```

Comment:
you could make this to shorten the code
```suggestion
    const [studentA, studentB] = order === 'asc'
      ? [a, b]
      : [b, a];
```

Reply:
Awesome